### ISSUE TITLE
Images not copied along with html file

### ISSUE THREAD

**Issue by sebelk** [Zettlr/Zettlr#644]
## Description

Embedded images are not copied along with hitml file. 

## Reproducing

1. Insert an image
2. Export to reveal.js
3. Go to browser, presentation will show image filename twice

## Expected behaviour

Image should be exported along the html file on the same directory

![imagen](https://user-images.githubusercontent.com/4109256/78393263-d589c280-75bf-11ea-95d6-d8f68a5ad91e.png)

A workaround is to copy manually images to export directory.

## Platform
 - OS and version: Fedora 31
 - Zettlr Version: 1.6.0
 - Screen Resolution: 1920 x 1080

**Comment by nathanlesage** [comment]
Yes, the reason is simple: HTML does not support embedding images\*, so when designing a presentation, you can choose two options:

1. If you will run the presentation on the same computer as you're writing it on, it's fine to simply use absolute paths and leave the export path to the temporary working directory, as the browser will find the images (due to them having absolute paths)
2. If you may be giving the presentation on another computer, make sure to only use relative paths and output the file to the working directory, because this way you can copy both the presentation **and** the folder containing the images.

In short: Zettlr does not transform your links when exporting to HTML, it leaves them as they are. It only transforms links for other types of exports (where some platforms do have problems as of now …)


\* _Well, technically it does support embedding images, but that will inflate the file size unnecessarily and may cause wild, unrelated issues._
